.gitignore

```diff
@@ -19,3 +19,6 @@ result-*
 
 # generated by pkgs/common-updater/update-script.nix
 update-git-commits.txt
+
+# VS Code workspace files
+.vscode/
```

Comment:
Could you remove this and send it in a different PR?

I imagine it is probably okay to add this, but I don't know anything about vscode, or if there are any rules here in nixpkgs about adding to the `.gitignore` file.